astropy/coordinates/earth.py

```diff
@@ -20,6 +22,9 @@
 # Available ellipsoids (defined in erfam.h, with numbers exposed in erfa).
 ELLIPSOIDS = ('WGS84', 'GRS80', 'WGS72')
 
+_NO_ONLINE_SITES_WARNING_MSG = ('Could not access the online site list. Falling'
```

Comment:
This is missing a space between "falling" and "back".

As a general policy when splitting a strong across lines I always add a space at the end of the line so that I don't forget.